source/database-connections/database-connections.txt

```diff
@@ -19,21 +19,42 @@ You can save this information as a database connection.
 Use Cases
 ---------
 
-You can save connection settings for relational and MongoDB databases
-when creating projects and migration jobs. You can use these saved database
-connections for any task that requires connecting to a relational or MongoDB
-database.
+You can save a new database connection:
```

Comment:
[Suggestion]

Our [style guide](https://www.mongodb.com/docs/meta/style-guide/style/lists/#introductory-text) suggests that we should avoid using a fragment to introduce a list. While what you have here is technically a full sentence, it might be slightly easier to parse if we write this as something like:

```suggestion
To save a new database connection, perform one of these actions:
```